ee/clickhouse/queries/breakdown_props.py

```diff
@@ -91,6 +101,22 @@ def get_breakdown_prop_values(
     )[0][0]
 
 
+def _to_value_expression(
+    breakdown_type: Union[Literal["event", "person", "cohort", "group"], None],
+    breakdown: Union[str, List[Union[str, int]], None],
+    breakdown_group_type_index: Optional[int],
+) -> str:
+    if breakdown_type == "person":
+        return get_single_or_multi_property_string_expr(breakdown, "person", "person_props", "value")
```

Comment:
non-blocking and off topic

Seems that for each table, the value of the property column i.e. "person_props" is always the same. Wonder if it's worth either inferring the property column from the table name, e.g. `get_single_or_multi_person_property_string_expr` or some lookup.